scripts/bb-build.sh

```diff
@@ -87,13 +98,18 @@ packages|kmod|pkg-kmod|dkms|dkms-kmod)
 	;;
 in-tree)
 	./configure $CONFIG_OPTIONS $LINUX_OPTIONS >>$CONFIG_LOG 2>&1 || exit 1
-	make $MAKE_OPTIONS >>$MAKE_LOG 2>&1 || exit 1
+	$MAKE $MAKE_OPTIONS >>$MAKE_LOG 2>&1 || exit 1
 	./scripts/zfs-tests.sh -cv >>$INSTALL_LOG 2>&1
 	sudo -E scripts/zfs-helpers.sh -iv >>$INSTALL_LOG 2>&1
 	;;
+freebsd)
```

Comment:
Nothing here looks to be platform specific, why don't we make this a generic install method called "system".